cni/pkg/iptables/testdata/default.golden

```diff
@@ -0,0 +1,16 @@
+iptables -t mangle -N ISTIO_PRERT
+iptables -t nat -N ISTIO_OUTPUT
+iptables -t mangle -N ISTIO_OUTPUT
+iptables -t mangle -A PREROUTING -j ISTIO_PRERT
+iptables -t mangle -A OUTPUT -j ISTIO_OUTPUT
+iptables -t nat -A OUTPUT -j ISTIO_OUTPUT
+iptables -t mangle -A ISTIO_PRERT -m mark --mark 0x3/0xfff -j CONNMARK --set-xmark 0x111/0xfff
```

Comment:
Where is it marked with 0x3

Reply:
ztunnel applies this mark to things it has proxied: 

https://github.com/istio/istio/pull/48253/files#diff-e979dbe9201b8c8f280cbe55ba0bb7ebf7c5aaf1e63134ecb697e794c9c538d1R36

See the sibling ztunnel PR: https://github.com/istio/ztunnel/pull/747/files#diff-99043dc51f03a8e0ce2a930635585be11a63b16088230ce30b31f2696cf48a6dR76